docs/section22/Rule22-17.md

```diff
@@ -0,0 +1,61 @@
+
+# CHW&CW - Rule 22-17  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** FALSE  
+**Rule ID:** 22-17  
+**Rule Description:** The baseline heat rejection device shall have an efficiency of 38.2 gpm/hp  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11  
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops, if so, continue to the rule logic: `if heat_rejection.loop.id in heat_rejection_loop_ids: CONTINUE TO RULE LOGIC`
+  
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+  
+  - check if the heat rejection has fan_shaft_power (not sure the exact python command for this): `if heat_rejection.fan_shaft_power:
+  
+    - set fan_shaft_power equal to heat_rejection.fan_shaft_power: `fan_shaft_power = heat_rejection.fan_shaft_power`
+    
+  - otherwise, we need to calculate shaft power based on the motor nameplate power: `else:`
+  
+    - set fan_shaft_power equal to the heat_rejection.fan_motor_nameplate_power * 0.9 (0.9 is a rule of thumb suggested by Michael Rosenberg in April 2023): `fan_shaft_power = heat_rejection.fan_motor_nameplate_power * 0.9`
+    
+  - calculate fan brake horsepower by multiplying the fan shaft power by the fan efficiency and 0.00134102 (the conversion from Watts to HP): `fan_BHP = fan_shaft_power * heat_rejection.fan_motor_efficiency * 0.00134102`
```

Comment:
Since fan_shaft_power is equal to BHP, we shouldn't be multiplying it by motor efficiency here. 
Also, I am not sure whether we need to conver to HP since that wuold be done by RCT as it parses the RMD riles. @weilixu please confirm.

So if fan_shaft_power is entered, than fan_BHP = fan_shaft_power; else if fan_motor_nameplate is 
entered then fan_BHP = fan_motor_nameplate_power * fan_motor_efficiency *0.9

Reply:
thanks, @mkarpman.  I've updated the fan_shaft_power (W) calculation per your notes.

I'm leaving the conversion to BHP for the moment until I have a chance to review it with @weilixu.